blog/2023-01-24-how-ai-creates-art/index.mdx

```diff
@@ -5,13 +5,14 @@ authors: [zain]
 date: 2023-01-24
 tags: []
 image: ./img/hero.png
+description: Machine learning models that can create beautiful and novel images. Learn how that works and how you could make use of it.
```

Comment:
Probably want to remove the "that" from "Machine learning models that can create beautiful and novel images."